integration/testrunner/db.go

```diff
@@ -24,37 +24,28 @@ import (
 )
 
 // queryIfAuditLogExists queries the DB and checks if audit log contained in the query exists or not.
-func queryIfAuditLogExists(ctx context.Context, tb testing.TB, query *bigquery.Query, expectedNum int64) (bool, error) {
+func queryIfAuditLogExists(ctx context.Context, tb testing.TB, query *bigquery.Query) ([]bigquery.Value, error) {
```

Comment:
Is it possible to return `[]*LogEntry` here instead? Basically converting each BQ row into a `*LogEntry`

Reply:
+1